feature-libs/cart/quick-order/components/cart-quick-order-form/cart-quick-order-form.component.ts

```diff
@@ -89,11 +89,21 @@ export class CartQuickOrderFormComponent implements OnInit, OnDestroy {
   }
 
   protected buildForm(): void {
+    const shouldHaveRequiredValidator = !this.featureConfig.isEnabled(
```

Comment:
I think we should have a TODO comment here with the ticket number for breaking change cleanup later.

Reply:
Changed